extmod/moducryptolib.c

```diff
@@ -69,6 +80,7 @@ struct mbedtls_aes_ctx_with_key {
     } u;
     unsigned char iv[16];
 };
+
```

Comment:
This change contradicts https://github.com/micropython/micropython/wiki/ContributorGuidelines , "The same applies to unrelated changes - for example, don't put formatting changes and actual code changes in one commit."

Reply:
Moved to a separate formatting commit

Reply:
Will remove https://github.com/micropython/micropython/pull/4727#discussion_r280585799